packages/story-editor/src/components/canvas/shoppingPageAttachment/index.js

```diff
@@ -90,7 +90,8 @@ const TextWrapper = styled.span`
 const LIGHT_COLOR = '#FFFFFF';
 const DARK_COLOR = '#000000';
 
-function PageAttachment({ theme }) {
+function PageAttachment({ pageAttachment = {} }) {
+  const { ctaText = __('Shop Now', 'web-stories'), theme } = pageAttachment;
```

Comment:
Related to my other suggestion to use `<ShoppingPageAttachment {...pageAttachment} />`

Instead of a single `pageAttachment` prop I think we can just keep separate `theme` and `ctaText` props. It's not like we're going to have tons of props here.

```suggestion
function PageAttachment({ ctaText = __('Shop Now', 'web-stories'), theme }) {
```